src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -465,6 +465,10 @@ const QuickMarcEditor = ({
     name: 'save',
     shortcut: 'mod+s',
     handler: async (e) => {
+      if (['ctrl', 'alt'].every(x => e.pressedKeys.includes(x))) {
```

Comment:
@przemyslawturek please add a comment explaining that ctrl+alt+S is a combination for `Ś`, so to prevent the saving we need to check for this combination

Reply:
ok

Reply:
This will block all `ctrl+alt+...` key combinations. Maybe this will fix the issue:
```
function isPolishDiacritic(char) {
  const polishDiacriticRegex = /[ąćęłńóśźżĄĆĘŁŃÓŚŹŻ]/;
  return polishDiacriticRegex.test(char);
}
```

Reply:
Another way is to check if the character is a diacritical character and if so, prevent the action.

Reply:
I think to detect diacritical characters is more universal for other languages

Reply:
Looks good, thanks.

Reply:
It may be worth revising this approach too https://github.com/folio-org/ui-inventory/pull/2350.